cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/ManagementClusterInventory.tsx

```diff
@@ -13,67 +13,84 @@ import { ManagementCluster } from '../../swagger-api/models/ManagementCluster';
 import { ManagementService } from '../../swagger-api';
 import { NavRoutes } from '../../shared/constants/NavRoutes.constants';
 import './ManagementClusterInventory.scss';
+import PageNotification, { Notification, NotificationStatus } from '../../shared/components/PageNotification/PageNotification';
 
 function ManagementClusterInventory() {
+    const [showLoading, setShowLoading] = useState<boolean>(false);
+    const [notification, setNotification] = useState<Notification | null>(null);
     const [managementClusters, setManagementClusters] = useState<ManagementCluster[]>([]);
     const [showDeleteModal, setShowDeleteModal] = useState<boolean>(false);
     const [clusterNameForDeletion, setClusterNameForDeletion] = useState<string>('');
 
     const navigate = useNavigate();
 
-    const retrieveManagementClusters = function () {
-        ManagementService.getMgmtClusters().then(
-            (data) => {
-                setManagementClusters(data);
-            },
-            (err) => {
-                console.warn(`management clusters get api failed with error: ${err}`);
-            }
-        );
-    };
+    // sets notification to null to dismiss alert
+    function dismissAlert() {
+        setNotification(null);
+    }
 
-    // Retrieve management clusters list on page load
+    // Retrieve management clusters list on page load; then set loading to false
     useEffect(() => {
-        retrieveManagementClusters();
+        retrieveManagementClusters().then(() => {
```

Comment:
please use async await. you may need to create the function inside the scope and call it.

Reply:
the async function returns a promise, so shouldn't using then be an acceptable way to consume it? It seems like it would be unnecessary to create the function inside this scope.